### PR TITLE
examples - php Websocket / watch method

### LINKED ISSUE #15350: How to use ccxt.pro to watch binanceusdm perp positions?
Hi, I can only see the watchTrades, but not watchPositions, how to watch binanceusdm perp?

### PR DESCRIPTION
**Pull request by ttodua**
1) I know that you might ask why I removed two other files, but I believe that same thing to be scattered across multiple example files are more time consuming and confusing to users. I think the example file I've added, optimizely demonstrates how to use watch method and also covers cases which were present in removed files. It will be a good "go-to-example" to link into issues and etc , and no need to point them to different multiple example files.

2) about caching - I have made a dedicated example for php "how to reload exchange instance from cache". So, that should be a separate file, without any much demonstration of ccxt-specific watch/fetch methods, as the programmers here will need specifically the way "how to efficiently reload exchanges from cache" and other example files should be meant for CCXT's watch/fetch methods demonstrations, and I believe they should better not to be heavily mixed together, so the sample `fetch-trades` call is enough there for demonstrational use.  
further note about "reloading of cached exchange" - the only fields that are missing from cache-reloaded exchange are: `lastRestRequestTimestamp, last_http_response, last_json_response, last_response_headers` and also `curl` is set to null instead of `CurlHandle::__set_state(array()),`. However, they doesn't seem to affect any functionality.

fix #15350

### CODE REVIEW

**Comment by ttodua** [comment]
@carlosmiei don't review this yet, i'll ping you later.

File: examples/ccxt.pro/php/many-exchanges-synchronously.php [hunk 1]
```diff
@@ -1,27 +0,0 @@
-<?php
```
Comment (carlosmiei) [inline examples/ccxt.pro/php/many-exchanges-synchronously.php:1]: @ttodua I've read the PR description but I'd argue we should not remove these examples, we have tons of them for the other languages and some of them are similar to each other, as long as they work properly there's no reason to ditch them

Comment (ttodua) [reply examples/ccxt.pro/php/many-exchanges-synchronously.php:1]: I think that having tons of similar examples are actually bad, but ofc, i'll revert them back.

Comment (ttodua) [reply examples/ccxt.pro/php/many-exchanges-synchronously.php]: I've reverted those files too, this PR is also ready.

**Comment by carlosmiei** [comment]
LGTM ✅

**Review by carlosmiei — APPROVED** [review]

**Pull request merged**